extensions/roc-plugin-test-jest/src/config/roc.config.js

```diff
@@ -0,0 +1,14 @@
+export default {
+    settings: {
```

Comment:
This structure works for me but as long as we have the same structure in all of the test plugins, the structure I used in this PR is the same as we have in the webdriver.io project. So we should set a best practice :D